node/src/accountant/test_utils.rs

```diff
@@ -320,6 +320,36 @@ impl AccountantBuilder {
         )
     }
 
+    pub fn sent_payable_dao(mut self, sent_payable_dao: SentPayableDaoMock) -> Self {
+        match self.sent_payable_dao_factory_opt {
+            None => {
+                self.sent_payable_dao_factory_opt =
+                    Some(SentPayableDaoFactoryMock::new().make_result(sent_payable_dao))
+            }
+            Some(sent_payable_dao_factory) => {
+                self.sent_payable_dao_factory_opt =
+                    Some(sent_payable_dao_factory.make_result(sent_payable_dao))
+            }
+        }
+
+        self
+    }
```

Comment:
Eventually, the other way of setup will need to be provided here. It must be rewritten. But I have taken care of it for at least one case of these in my card, co I dunno. Maybe you can keep it and I will do a clean-up during the merge.

Because it is possible that I will have the PR ready before you, I suggest you to write a todo to the both of these blocks of code (`sent_payable_dao` and `failed_payable_dao`) so that you will recall that you must opt for the different setup when you're doing the merge. Don't keep this. We need to be able to specify to which DAOs the factories are meant because there will be eventually two resorts. The PendingPayableScanner and PayableScanner.